rollup/internal/controller/relayer/relayer_test.go

```diff
@@ -25,8 +26,8 @@ var (
 
 	base *docker.App
 
-	// l2geth client
-	l2Cli *ethclient.Client
+	// geth client
+	l1Cli *ethclient.Client
```

Comment:
why changing to `l1Cli`?

Reply:
Because L2 relayer uses L1 client to connect to the L1 node.

Reply:
code snippet:
```
	// Create geth client.
	l1Cli, err = base.L1Client()
	assert.NoError(t, err)
```

```
// L1Client returns a ethclient by dialing running l1geth
func (b *App) L1Client() (*ethclient.Client, error) {
	if utils.IsNil(b.L1gethImg) {
		return nil, fmt.Errorf("l1 geth is not running")
	}
	client, err := ethclient.Dial(b.L1gethImg.Endpoint())
	if err != nil {
		return nil, err
	}
	return client, nil
}
```

Reply:
update: I see, I'll revert the change and description.

Reply:
For L1 layer, the message fetcher client connects to L1 service provider, and the sender connects to L2 service provider.